lib/rspec/matchers/built_in/captures.rb

```diff
@@ -0,0 +1,78 @@
+module RSpec
+  module Matchers
+    module BuiltIn
+      # @api private
+      # Provides the regex with captures matcher
+      class Captures < BaseMatcher
```

Comment:
Actually I think that's a good improvement, it would eliminate the need to instantiate it at all, you could just call `super`